src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/LocalSlotMappingTests.cs

```diff
@@ -2965,6 +2965,7 @@ .locals init (bool V_0)
         [Fact]
         public void For()
         {
+            try {
```

Comment:
>try { [](start = 12, length = 5)

Did you intend to leave this instrumentation in?

Reply:
No. Thanks for catching.